client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/rpc/schema/params/ByteBufferParam.java

```diff
@@ -62,6 +62,7 @@ protected void setValueBasedOnValidInstance(Object instance) {
 
     @Override
     public void setValueBasedOnInstanceOrJson(Object json) throws JsonProcessingException {
+        if (json == null)  return;
         assert json instanceof String;
```

Comment:
`assert` would be used for `private` methods, to assert something that should never happen, otherwise it is a bug.
but, for a `public`, it should rather throw a `IllegalArgumentException`. Recall [this discussion](https://github.com/EMResearch/EvoMaster/blob/master/docs/for_developers.md#pre-and-post-conditions).

Reply:
thanks! ll update them and request a review once tests pass